vdev/src/testing/integration.rs

```diff
@@ -65,6 +65,19 @@ impl IntegrationTest {
             env_vars.insert(key, Some(value));
         }
         let mut args = self.config.args.clone();
+
+        args.push("--features".to_string());
+        args.extend(self.config.features.clone());
```

Comment:
Note that the feature names need to be joined with a `,`, so this should be:
```suggestion
        use itertools::Itertools as _; // at the top of the file
        args.extend(self.config.features.join(","));
```

Reply:
I committed the changes with only one modification of using args.push instead of args.extend since after the .join we end up with a String